### DISCUSSION TITLE
Style tooltip arrow in theme styles

### DISCUSSION THREAD

**Discussion by jinsley8** [chakra-ui/chakra-ui#6834] [Q&A] [answered]
Does anyone know how to style the Tooltip arrow from a custom theme with the baseStyle?

This is what I have:
```
export const tooltipStyles = {
    components: {
        Tooltip: {
            baseStyle: {
                color: '#F6F6F6',
                fontFamily: 'heading',
                fontSize: 'sm',
                fontWeight: 500,
                lineHeight: '20px',
                letterSpacing: '0.25px',
                borderRadius: '4px',
                bg: '#56595C',
                padding: '8px 12px',
            },
        },
    },
};
```

I'm just trying to change the arrow background color here.

**Comment by jukebox42** [comment, marked as answer]
Just fumbled on the solution for this one looking through the [tooltip source](https://github.com/chakra-ui/chakra-ui/blob/main/packages/components/theme/src/components/tooltip.ts#L6). You can override the css variable for it.

```diff
 export const tooltipStyles = {
     components: {
         Tooltip: {
             baseStyle: {
+                "--popper-arrow-bg": "#F00",
                 color: '#F6F6F6',
                 fontFamily: 'heading',
                 fontSize: 'sm',
                 fontWeight: 500,
                 lineHeight: '20px',
                 letterSpacing: '0.25px',
                 borderRadius: '4px',
                 bg: '#56595C',
                 padding: '8px 12px',
             },
         },
     },
 };
```

It looks like all cssVar is doing is prepending `--` to the string so I just hand wrote it. I presume if you want to be future proof you could import and use it like they do. If you want to access color variables you need to include `colors.` as part of the value. ex: `"--popper-arrow-bg": "colors.brand.200",`
